index.html

```diff
@@ -316,6 +316,10 @@ <h3>
           <li>Let |document:Document| be [=this=]'s [=relevant global
           object=]'s [=associated Document=].
           </li>
+          <li data-tests="wakelock-active-document.https.window.js">If
```

Comment:
Is this a duplicate of lines [334-337](https://github.com/w3c/screen-wake-lock/pull/367/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R334-R337)?

Reply:
lol yes. But the fully active check should be done first.